core/main/src/bootstrap/extn/load_session_step.rs

```diff
@@ -42,14 +42,6 @@ impl Bootstep<BootstrapState> for LoadDistributorValuesStep {
             return Ok(());
         }
         MainContextProcessor::initialize_session(&s.platform_state).await;
-        // Session available during startup make sure the context is updated with the token for first time
```

Comment:
Is this being done elsewhere now ripple initialization? We have new changes for DistP initialization that depend on receipt of a session context update on start-up.

Reply:
+1 we need this for cases where token is available during startup

Reply:
Yes, this state will be updated [here](https://github.com/rdkcentral/Ripple/blob/c3e23fd9668722fd3a7337e12d49595389151fc4/core/main/src/processor/main_context_processor.rs#L93).